canonical-web-and-design/deployment-configs
# Consolidate global config for services

- Add TALISKER_NETWORKS, so we can see metrics when they exist
- Add prometheus.io annotations, so we can gather metrics when needed
- Apply global environment variables to all services
- Hard-code SENTRY_DSN values, as they aren't secret any more

QA
--

We'll now be applying the same HTTP_PROXY settings to all services,
so it's imperative that we check this works with any sites that might be
making API calls.

To my knowledge, the ones this might effect worst might be jujucharms.com
and jaas.ai. But we should look through the list and check any that look important.
